independent-publisher-connectors/GitLab/apiDefinition.swagger.json

```diff
@@ -0,0 +1,703 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v4",
+    "title": "GitLab",
+    "description": "A connector for the GitLab REST API.",
+    "contact": {
+      "name": "Roy Paar",
+      "email": "roypaar@microsoft.com"
+    }
+  },
+  "host": "gitlab.com",
+  "basePath": "/api/v4",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/version": {
+      "get": {
+        "description": "Retrieve version information for this GitLab instance.",
+        "summary": "getVersion",
+        "tags": [
+          "version"
+        ],
+        "operationId": "getVersion",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/VersionResponse"
+            },
+            "examples": {
+              "application/json": {
+                "version": "13.3.0-pre",
+                "revision": "f2b05afebb0"
+              }
+            },
+            "headers": {}
+          },
+          "401": {
+            "description": "unauthorized operation",
+            "headers": {}
```

Comment:
@RoyPaarMS Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Removed